pkg/apis/api.acorn.io/v1/region.go

```diff
@@ -0,0 +1,52 @@
+package v1
+
+import (
+	v1 "github.com/acorn-io/acorn/pkg/apis/internal.acorn.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const RegionConditionClusterReady = "ClusterReady"
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type Region struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	Spec   RegionSpec   `json:"spec,omitempty"`
+	Status RegionStatus `json:"status,omitempty"`
+}
+
+type RegionSpec struct {
+	Description string `json:"description,omitempty"`
+	AccountName string `json:"accountName,omitempty"`
```

Comment:
Don't have the full context, Just want to call out that accountName is hub specific. Are we ok to make it into open source types?

Reply:
Fixed.